ts/output/common/Wrappers/mrow.ts

```diff
@@ -181,7 +181,8 @@ export function CommonMrowMixin<
         //  Stretch the stretchable children
         //
         for (const child of stretchy) {
-          child.coreMO().getStretchedVariant([H, D]);
+          const rscale = child.coreRScale();
+          child.coreMO().getStretchedVariant([H / rscale, D / rscale]);
```

Comment:
Compute a constant before the loop rather than performing the division every time.

Reply:
The division is by a potentially different value of `rscale` for each child, so can't be computed ahead of time.